docs/documenting-schema.rst

```diff
@@ -0,0 +1,73 @@
+.. _documenting-schema:
+
+Documenting schema
+=======================
+
+You can improve the experience of consuming your GraphQL API by documenting the types that it supports.
+
+Keeping your documentation close to the code that supports it is a great way to ensure that it is always accurate and up-to-date. To support this workflow, GraphQL allows programmers to write descriptions directly alongside types in `GraphQL Schema Definition Language syntax <https://facebook.github.io/graphql/June2018/#sec-Descriptions>`_.
+
+
+Writing Descriptions
+--------------------
+
+GraphQL Descriptions are declared using a python-like docstring format, and may include Markdown features::
+
+    query = '''
+        """
+        A simple GraphQL schema which is well described.
+        """
+        type Query {
+            "A list of all users of the service."
+            users: [User]!
+        }
+    '''
+
+    user = '''
+        """
+        A user is an individual's account.
+        """
+        type User {
+            "Unique ID of the object."
+            id: ID!
+
+            "The name the user is identified by in the service."
+            username: String!
+
+            "The date this user object was created."
+            joinedOn: Datetime!
+
+            "The user's birth data."
+            birthDay: Date!
+        }
+    '''
+
+
+Introspection
+-------------
+
+GraphQL Descriptions defined this way will become available via a GraphQL introspection query. This will also permit interactive GraphQL API explorers to include your descriptions in their interactive documentation::
+
+    query IntrospectionQuery {
+        __schema {
+            types {
+                kind
+                name
+                description
+            }
+        }
+    }
+
+.. code-block:: json
```

Comment:
We probably could add a little comment here.

Reply:
Yeah?  Maybe something just noting that this is an illustration of the response payload?

There are a few other places in the docs that use this kind of illustration already, so I thought we might be able to get away with it here.  But you're probably right, it wouldn't hurt to add.

https://ariadne.readthedocs.io/en/0.2.0/scalars.html

Reply:
Yeah, just a little caption.

Reply:
I prefer paragraphs to in-code comments. Speaking from experience the approach of having few laser-focused code bits with well-described text around them produces better docs.